pt-BR/active_record_migrations.md

```diff
@@ -399,82 +399,81 @@ change_table :products do |t|
 end
 ```
 
-removes the `description` and `name` columns, creates a `part_number` string
-column and adds an index on it. Finally it renames the `upccode` column.
+remove as colunas `description` e `name`, cria uma coluna de string `part_number`
+e adiciona um índice nela. Finalmente renomeia a coluna `upccode`.
 
-### Changing Columns
+### Mundado Colunas
 
-Like the `remove_column` and `add_column` Rails provides the `change_column`
-migration method.
+Assim como o `remove_column` e `add_column`, o Rails fornece o método de
+*migration* `change_column`.
 
 ```ruby
 change_column :products, :part_number, :text
 ```
 
-This changes the column `part_number` on products table to be a `:text` field.
-Note that `change_column` command is irreversible.
+Isso muda a coluna `part_number` na tabela produtos para ser um campo `:text`.
+Note que o comando `change_column` é irreversível.
 
-Besides `change_column`, the `change_column_null` and `change_column_default`
-methods are used specifically to change a not null constraint and default
-values of a column.
+Além do `change_column`, os métodos `change_column_null` e `change_column_default`
+são usados especificamente para alterar uma *not null constraint* e valores
```

Comment:
O que acha de traduzirmos esse `not null constraint` para algo como `restrição de não ser nulo`, fica ruim?

O que acha @campuscode/guia-rails-team alguém com uma ideia?

Reply:
> O que acha de traduzirmos esse `not null constraint` para algo como `restrição de não ser nulo`, fica ruim?
> 
> O que acha @campuscode/guia-rails-team alguém com uma ideia?

Concordo.